examples/c5g7_transient/rod_translation/settings.py

```diff
@@ -0,0 +1,31 @@
+import openmc
```

Comment:
Should this file be removed? It looks like settings are already defined in c5g7-3d-mg-rod.py